manufacturing/filter_holder/README.md

```diff
@@ -42,13 +52,15 @@ We may have parts to send you from our inventory.
 | 6    | ~4 sq in | McMaster-Carr | Gasket Material      | $22.22/ 12x12" sheet | [C][9mcmc] | cut with jigs to make items 4 and 5 |
 | 7    |        2 | McMaster-Carr | 96016A556            | $5.50/10pk       | [C][7mcmc] | thumbscrews |
 | 8    |        2 | McMaster-Carr | 94180A333            | $16.69/100pk     | [C][8mcmc] | thumbscrews |
+| 9    |        1 | RespiraWorks  | Filter Non-patient Outer Housing | $1 (material)    | [B][9rw]   | 3D printed from data in link |
```

Comment:
I would move this to right after items 1 and 2.

Reply:
And the quantities should probably be reflected for the entire ventilator. So 2x of these, and 2x of 1, 4x of 2? Does that make sense?

Reply:
yeah it make sense and will update that.